src/index.html

```diff
@@ -7,6 +7,373 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartfones" class="nav__link">
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+              >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <div class="content__product-card product-card" data-qa="card">
+        <img
+          class="product-card__img"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+
+        <h2 class="product-card__description">
+          APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code reset">Product code: 195434</p>
```

Comment:
remove unused classes
```suggestion
        <p class="product-card__code">Product code: 195434</p>
```